### PR TITLE
Pivot tables – Add "Show totals" column setting

### LINKED ISSUE #14172: Pivot tables: grouping field settings
This is missing functionality per the [product doc](https://docs.google.com/document/d/1ScNpNjjYlsi6syyHFKctTlZ149g5ulTIHFnAmmq6h58/edit#). Pivot tables are supposed to have a way to show settings for a row or column grouping field to allow the user to:
- Turn subtotals off/on
- Change the sort order for that group
- Access the column formatting options for the values in that group

image

### PR DESCRIPTION
**Pull request by paulrosenzweig**
Resolves #14172

This is the final part of that issue. The other parts were in #14559 and #14464.

This PR adds another column setting in pivot tables. That setting only appears for columns that have subtotals and defaults to `true`. Toggling it to `false` hides all subtotals. If any of those now subtotal-less items were collapsed, we uncollapse them.

showtotals

### CODE REVIEW

File: frontend/src/metabase/visualizations/components/settings/ChartSettingFieldsPartition.jsx [hunk 1]
```diff
@@ -30,104 +31,90 @@ const DragWrapper = styled.div`
...
-        onClick={() => onChange(isSelected ? undefined : buttonValue)}
-        size={16}
-        className={cx("sort cursor-pointer", {
-          "text-brand": isSelected,
-          "text-medium text-brand-hover": !isSelected,
-        })}
-      />
-    );
-  }
+function SortButton({ iconName, onChange, currentValue, buttonValue }) {
```
Comment (paulrosenzweig) [inline frontend/src/metabase/visualizations/components/settings/ChartSettingFieldsPartition.jsx:46]: The class->function switches here are unrelated to the change. The component above no longer needed state, so I switched them all over to match.

**Comment by mazameli** [comment]
Nice! Shouldn't this target `release-x.38.x`?

**Review by kdoh — APPROVED** [review]

**Review by ariya — APPROVED** [review]
LGTM :+1:

**Comment by paulrosenzweig** [comment]
@nemanjaglumac the failure here is unrelated to the changes, right? Any objections to merging this?

**Comment by flamber** [comment]
@paulrosenzweig Unrelated - we're fighting with that now, so just ignore

**Pull request merged**